docs/nddata/utils.rst

```diff
@@ -0,0 +1,257 @@
+.. _nddata_utils:
+
+astropy.nddata.utils
+====================
+
+Overview
+--------
+
+The `astropy.nddata.utils` module includes general utility functions
+for array operations.
+
+
+Cutout Images
+-------------
+
+Getting Started
+^^^^^^^^^^^^^^^
+
+The `~astropy.nddata.utils.Cutout` class can be used to create a
```

Comment:
This `Cutout` class only works with 2D images (I think), but is located in `nddata`.
Probably it's not easy to make it work nicely or at all for ND?
So maybe leave implementation as-is, but rename to `CutoutImage` or `Cutout2D`?
(so that in the future someone can implement the equivalent 1D or 3D or ND functionality ...